app/scripts/modules/core/src/managed/ManagedReader.ts

```diff
@@ -89,7 +88,7 @@ export class ManagedReader {
       .then(this.decorateResources);
   }
 
-  public static getEnvironmentsSummary(app: string): PromiseLike<IManagedApplicationEnvironmentSummary> {
```

Comment:
@erikmunson I think this refactor could have exposed a typing problem.  Can you confirm that I did the right thing here?